tests/tests.go

```diff
@@ -353,6 +382,36 @@ func (ctl *controller) patch(shortURL string, body io.Reader) (io.ReadCloser, er
 	return resp.Body, nil
 }
 
+func (ctl *controller) reachHealthz() error {
```

Comment:
nit: name it waitHealthz for consistency.

Reply:
SG.